src/zjs_ocf.json

```diff
@@ -14,34 +15,27 @@
         "common": ["zjs_ocf_common.c"]
     },
     "zjs_config": ["-DBUILD_MODULE_OCF"],
+    "virtualdeps": ["net-l2"],
     "zephyr_conf": {
         "all": [
             "CONFIG_NETWORKING=y",
             "CONFIG_NET_IPV6=y",
             "CONFIG_NET_UDP=y",
             "CONFIG_TEST_RANDOM_GENERATOR=y",
             "CONFIG_INIT_STACKS=y",
-            "CONFIG_NET_NBUF_RX_COUNT=14",
-            "CONFIG_NET_NBUF_TX_COUNT=14",
+            "CONFIG_NET_BUF=y",
+            "CONFIG_NET_PKT_RX_COUNT=5",
```

Comment:
These are the defaults from the echo server sample in Zephyr. I guess an alternative would be to leave them out entirely and take whatever defaults there are.

But the lines above are no longer valid so these were not actually being set at all.